app/controllers/member/details_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Member::DetailsController < ApplicationController
+  include MemberConcerns
+  include MailingListConcerns
+
+  before_action :set_member
+  before_action :suppress_notices
+
+  def edit
+    accept_terms
+
+    flash[notice] = I18n.t('notifications.signing_up')
+    @member.newsletter ||= true
```

Comment:
@despo can you expand on what this line does? I'm not quite sure why this is here.

Reply:
It checks the newsletter checkbox by default. If they uncheck then they wont be added to the newsletter, if they leave it as is then they will be subscribed.

Reply:
@despo ok, it's what I thought but wanted to double check.